src/scripts/tell_me_about_username.script.ts

```diff
@@ -0,0 +1,40 @@
+import { RobotWithClient } from '../hackbot';
+
+export default (robot: RobotWithClient) => {
+
+  robot.respond(/tell me about @([a-z0-9.\-_]+)/i, response => {
+    const username = response.match[1];
+
+    const dataStore = robot.adapter.client.rtm.dataStore;
+    const user = dataStore.getUserByName(username);
+
+    if (user === undefined) {
+      response.reply(`"${username}" is not a user I recognise!`);
+      return;
+    }
+
+    robot.client.getUser(user.id)
+      .then(res => {
+        if (!res.ok && res.statusCode === 404) {
+          return response.reply(`"${user.name}" is not a user I recognise!`);
+        }
+
+        if (!('name' in res.user.team)) {
```

Comment:
So [this](https://github.com/TechNottingham/hubot-hackbot/blob/master/src/scripts/tell_me_about_my_team.script.ts#L10) is also broken?